homeassistant/components/recorder/__init__.py

```diff
@@ -90,25 +91,29 @@
 
 CONFIG_SCHEMA = vol.Schema(
     {
-        vol.Optional(DOMAIN, default=dict): FILTER_SCHEMA.extend(
-            {
-                vol.Optional(CONF_PURGE_KEEP_DAYS, default=10): vol.All(
-                    vol.Coerce(int), vol.Range(min=1)
-                ),
-                vol.Optional(CONF_PURGE_INTERVAL, default=1): vol.All(
-                    vol.Coerce(int), vol.Range(min=0)
-                ),
-                vol.Optional(CONF_DB_URL): cv.string,
-                vol.Optional(CONF_COMMIT_INTERVAL, default=1): vol.All(
-                    vol.Coerce(int), vol.Range(min=0)
-                ),
-                vol.Optional(
-                    CONF_DB_MAX_RETRIES, default=DEFAULT_DB_MAX_RETRIES
-                ): cv.positive_int,
-                vol.Optional(
-                    CONF_DB_RETRY_WAIT, default=DEFAULT_DB_RETRY_WAIT
-                ): cv.positive_int,
-            }
+        vol.Optional(DOMAIN, default=dict): vol.All(
+            cv.deprecated(CONF_PURGE_INTERVAL, invalidation_version="0.111.0"),
```

Comment:
Let's drop the `invalidation_version`. Let it be a warning and we can drop it at some point in the future. No reason to break things if we can keep it as a warning and it's not impacting operation.

Reply:
Done. Thank you for reviewing!